src/identity/quartzOrganizations/actions/thunks/index.ts

```diff
@@ -13,73 +16,95 @@ import {
 import {PublishNotificationAction} from 'src/shared/actions/notifications'
 
 // Types
-import {GetState, RemoteDataState} from 'src/types'
+import {AppThunk, GetState, RemoteDataState} from 'src/types'
 import {OrganizationSummaries} from 'src/client/unityRoutes'
-
 type Actions = QuartzOrganizationActions | PublishNotificationAction
 type DefaultOrg = OrganizationSummaries[number]
 
-class OrgNotFoundError extends Error {
+interface UpdateOrgParams {
+  accountId: number
+  newDefaultOrg: DefaultOrg
+}
+
+// Utils
+import {reportErrorThroughHoneyBadger} from 'src/shared/utils/errors'
+
+export enum ThunkErrorNames {
```

Comment:
`ThunkErrorNames` is fairly generic. Can you think of anything more specific and descriptive for this? I was thinking something like `OrganizationNetworkErrors`, or something along those lines, since it describes more specifically what these are.

Reply:
Good point - I'll rename the export something more descriptive that's related to `Organizations` specifically.